CWE/events/italy.txt

```diff
@@ -54,460 +126,341 @@ country_event = {
 		}
-		ai_chance = { factor = 0.8 }
-	}
-
-	option = {
-		name = EVT_8007734_B
-		any_pop = { militancy = 1.0 }
-		set_country_flag = radical_ICP 
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-		ai_chance = { factor = 0.2 }
+                scaled_militancy = { factor = 3 ideology = communist }
+			}	
+		any_pop = {
+                scaled_militancy = { factor = 3 ideology = communist_social }
+			}	
+        any_pop = {
+                scaled_militancy = { factor = 3 ideology = socialist }
+			}		
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = communist }
+			}	
+		any_pop = {
+                scaled_consciousness = { factor = 2 ideology = communist_social }
+			}	
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = socialist }
+			}
+		ai_chance = { factor = 0.9 }
 	}
 	
-}
-
-#Italian Popular front
-country_event = {
-  id = 8007735
-  title = EVT_8007735_NAME
-  desc = EVT_8007735_DESC
-  picture = "nwo2_italian_popular_front"
-  fire_only_once = yes
-
-	trigger = {
-		tag = ITA
-		year = 1947 NOT = { year = 1949 }
-		has_country_flag = moderate_ICP
-		OR = { government = democracy government = hms_government }
-	}
-
-	mean_time_to_happen = { months = 2 }
-
 	option = {
-		name = EVT_8007735_A
-		ai_chance = { factor = 0.8 }
-		any_pop = { consciousness = 5 }
+		name = "...or must it?"
+		
+		prestige = -50
+		
+		badboy = 3
+		
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -400 }
+        }
+		
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       diplomatic_influence  = { who = THIS value = -200 }
+        }
+		
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
-		set_country_flag = very_moderate_ICP 
-	}
-
-	option = {
-		name = EVT_8007735_B
-		ai_chance = { factor = 0.2 }
-		any_pop = { militancy = 3 }
+                scaled_militancy = { factor = 2 ideology = conservative }
+			}	
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-		set_country_flag = very_radical_ICP 
+                scaled_militancy = { factor = 2 ideology = big_tent }
+			}	
+        any_pop = {
+                scaled_militancy = { factor = 2 ideology = liberal }
+			}		
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = conservative }
+			}	
+		any_pop = {
+                scaled_consciousness = { factor = 2 ideology = big_tent }
+			}	
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = liberal }
+			}
+		ai_chance = { factor = 0.1 }
 	}
+   
 }
 
-#Togliatti assassination attempt
+#1948 Constitution
```

Comment:
Constitution event must be triggered immediately by the republic formation event because otherwise the generic event to choose republic type will fire in between. Alternative is to set Italy as a parliamentary republic in the ITA republic formation event first to block the generic event then this event just chooses whether to leave it as so or to pick another republic type, example presidential republic.